src/config.cc

```diff
@@ -180,6 +180,16 @@ namespace detail {
 			const auto result = get_env("CELERITY_FORCE_WG");
 			if(result.first) { CELERITY_WARN("Support for CELERITY_FORCE_WG has been removed with Celerity 0.3.0."); }
 		}
+
+		// -------------------------------- CELERITY_DRY_RUN ---------------------------------
+		{
+			const auto result = get_env("CELERITY_DRYRUN_NODES");
+			if(result.first) {
+				dry_run = std::stoi(result.second) > 0;
+				dry_run_nodes = std::stoi(result.second);
+				CELERITY_WARN("Celerity Dry Run detected.");
```

Comment:
As this is not a validation warning/error, I would suggest moving it to the runtime constructor! Also maybe rephrase to `"Performing a dry run with {} simulated nodes."`.